src/App.tsx

```diff
@@ -1,9 +1,17 @@
 import { NextUIProvider } from "@nextui-org/react";
+import HomePage from "./pages/HomePage";
+import ComicBreakdownPage from "./pages/ComicBreakdownPage";
+import SearchPage from "./pages/SearchPage";
+import Chapters from "./elements/Chapters";
+import MangaReaderPage from "./pages/MangaReaderPage";
+import GenreList from "./elements/GenresList";
 
 function App() {
   return (
     <NextUIProvider>
-      <></>
+      <>
```

Comment:
Fragments are not needed for one nested component. Only for multiple sibling components